MountSionCBSSecondary/SchoolWebsite
# Adding verification steps readme

# Description

Tutorial on how to verify changes on a website using local server has been added to the docs folder and linked to README.md

## What

New file verification-steps.md has been created and README.md has a new line to reflect this.

## Why

To allow new developers to have a tutorial to learn how to set up a local server for verification purposes.

## How

Changes were made using Markup and Sublime text and pushed up via terminal.

## Verification Steps

View README.md and ensure new link takes you to the new tutorial.
View verification-steps.md and ensure all steps are working correctly.


# Checklist:
